content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -426,12 +524,25 @@ same root of trust.
 
 ## Uninstalling the remote cluster
 
-To uninstall the remote cluster run the following command:
+To uninstall the remote cluster, run the following command:
 
 {{< text bash >}}
-$ istioctl --context=${REMOTE_CLUSTER_CTX} manifest generate <your original remote configuration> | \
-    kubectl delete -f -
+$ istioctl --context=${REMOTE_CLUSTER_CTX} manifest generate -f istio-remote0-cluster.yaml | \
+    kubectl delete -f - --context=${REMOTE_CLUSTER_CTX}
 
 $ istioctl x create-remote-secret --context=${REMOTE_CLUSTER_CTX} --name ${REMOTE_CLUSTER_NAME} | \
     kubectl delete -f - --context=${MAIN_CLUSTER_CTX}
+
```

Comment:
The `istioctl x create-remote-secret --context=${REMOTE_CLUSTER_CTX} --name ${REMOTE_CLUSTER_NAME} | \
    kubectl delete -f - --context=${MAIN_CLUSTER_CTX}` yields:
```
error: could not get access token to read resources from local kube-apiserver: serviceaccounts "istio-reader-service-account" not found
No resources found
```